### ISSUE TITLE
Failure to setup striker

### ISSUE THREAD

**Issue by rlsf** [stryker-mutator/stryker-js#3007]
**Summary**

trying to setup stryker on a project i'm working on.
worked according to manual:
```
npm install --save-dev @stryker-mutator/core
npx stryker init
```
results in the following error message:
```
Cannot find module 'mutation-testing-report-schema/api'
Require stack:
- /myProject/node_modules/@stryker-mutator/api/dist/src/core/mutant.js
- /myProject/node_modules/@stryker-mutator/api/dist/src/core/index.js
- /myProject/node_modules/@stryker-mutator/api/core.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/di/build-main-injector.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/di/index.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/process/1-prepare-executor.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/process/index.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/stryker.js
- /myProject/node_modules/@stryker-mutator/core/dist/src/index.js
- /myProject/node_modules/@stryker-mutator/core/bin/stryker
```

as a result, i can't generate stryker config/log/environment reports.

tried to manually install `mutation-testing-report-schema`: 
```
npm i --save-dev mutation-testing-report-schema
```
didn't fix the issue.

**Stryker config**

```json
{
}
```

**Test runner config**

**Stryker environment**

```
+-- @stryker-mutator/core@5.2.2
```

```
+-- mocha@9.0.2
```

**Test runner environment**

```shell
# Test command
```

```json
```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | 12.14.1
| npm              | 6.14.11
| Operating System | MacOS (Catalina, 10.15.7)

**Add stryker.log**

**Comment by nicojs** [comment]
This is very strange. Could you try to run `npm install` or `yarn install` again? It should install the correct version of muation-testing-elements

**Comment by rlsf** [comment]
> This is very strange. Could you try to run `npm install` or `yarn install` again? It should install the correct version of muation-testing-elements

tried, also tried to completely remove my `node_modules` and `package-lock.json` files and do npm on clean environment, same result.
also verified that i have only one copy of `mutation-testing-report-schema` in`node_module`.
the project is written in native javascript (no ts), could this be a problem?

**Comment by nicojs** [comment]
Could you try to install node 12.22.3 (latest V12). Pretty sure it has to do with the node version

**Comment by rlsf** [comment]
indeed, upgrading to node `v12.22.3` fixed the issue.
is the minimal node version documented somewhere? i couldn't find it in the readme/docs.

**Comment by nicojs** [comment]
Not explicitly, but we're only supporting LTS version of node. Which is `v12.22.3` at the moment, see https://nodejs.org/en/about/releases/

Since Stryker 5.2.x we're using `mutation-testing-report-schema@v1.7.4`, which adds exports using the "package export subpath pattern" feature, see https://nodejs.org/api/packages.html#packages_subpath_patterns. This is supported since node v12.20.

We should fix this, since using the subpath pattern of mutation-testing-report-schema is optional, we can simply not use it. We should then also run a build server on node 12.0.0. Either that, or we should clearly drop support for node 12.22.

**Comment by rlsf** [comment]
thanks, should i keep this ticket open for this fix or will you open a new one for it?

**Comment by nicojs** [comment]
No, it's fine to keep this one open.

**Comment by nicojs** [comment]
This feature just got released for Stryker 5.3 please give it a try
